stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Richard Choi</h4>
+            <p>DevRel Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/choir27" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/richard-choir/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Ryan Furrer</h4>
+            <p>UX Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/ryandotfurrer" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/ryanfurrer/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Walter Furrer</h4>
+            <p>Documentation Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/FurrerW" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/furrerw/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Shashi Lo</h4>
+            <p>Engineering Manager</p>
+          </div>
+          <div>
+            <a href="https://github.com/shashilo" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/shashilo/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Corina Murg</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/CorinaMurg" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/corinamurg/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Chris Nowicki</h4>
+            <p>Software Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/chris-nowicki" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/chris-nowicki/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
+        <div class="team-member flex flex-col gap-2">
+          <div>
+            <h4>Mai Vang</h4>
+            <p>Frontend Engineer</p>
+          </div>
+          <div>
+            <a href="https://github.com/vmaineng" target="_blank" rel="noreferrer">GitHub</a>
+            <a href="https://www.linkedin.com/in/mai-vang-swe/" target="_blank" rel="noreferrer">LinkedIn</a>
+          </div>
+        </div>
```

Comment:
feels like we could automate this a bit with a map array, where we have a list of objects with the team member name, their role, and their links. It also makes it easier to add members.  Also are we presenting the members in a particular order, like in alphabetical order or by role?

Reply:
Going to refer to my comment here: https://github.com/LetsGetTechnical/gridiron-survivor/pull/543#discussion_r1765243921

As far as presenting in a particular order, I went with alphabetical (last name). I am open to suggestions for the order, though.

Reply:
I like alphabetical, but I would be open to other suggestions.

Reply:
I have this set as alphabetical, other than Shashi. Shashi will always appear first in the array.